fbw-common/src/systems/instruments/src/EFB/Ground/Pages/Payload/A320_251N/A320Payload.tsx

```diff
@@ -140,10 +140,12 @@ export const A320Payload: React.FC<A320Props> = ({
         COMPLETED: 6,
     };
 
+    const generalBoardingInProgress = () => (gsxPayloadSyncEnabled ? gsxInProgress() : boardingStarted); // TODO this should be a Lvar and written in rust
```

Comment:
Remove the TODO or reexamine this general boarding var.

On review now that I've had a better look I really don't like this, the rust backend has been made with the concept that GSX has its own seperate hook via the payload layer to the MSFS payload values, we don't manage this at all.

The addition of the GSX vars here was always a temporary solution, meant to be replaced by a dedicated GSX frontend. (See https://github.com/flybywiresim/aircraft/pull/8225)